packages/modules/fronius/counter_s0.py

```diff
@@ -29,9 +29,8 @@ def update(self, bat: bool) -> CounterState:
         primo = self.component_config["configuration"]["primo"]
         log.MainLogger().debug("Komponente "+self.component_config["name"]+" auslesen.")
 
-        response = requests.get(
+        response = req.get_http_session().get(
```

Comment:
Es wäre sinnvoll die Session zu recyceln, wenn wie hier mehrere Requests an ein und den selben Host gehen. Also so:
```python
session = req.get_http_session()
response = session.get("...")
other_response = session.get("...")
```
Also in dem konkreten Fall geht hier (Zeile 32) ein Request raus und weiter unten in Zeile 46 noch ein Request. Wenn beide Requests das selbe `Session`-Objekt verwenden, dann gibt es den kleinen Performancevorteil des Connection-Poolings.